### PR TITLE
32 - adds further installation instructions

### LINKED ISSUE #32: Explicit installing instructions?
I am trying to install `hugo extended` in Ubuntu 20.04x64 using https://gohugo.io/getting-started/installing but it is not a straightforward process, perhaps I will try to add few instructions for those GNU/Linux users who want to have a go to the website.

### PR DESCRIPTION
**Pull request by mxochicale**
# Resolves #32

### CODE REVIEW

**Comment by NickleDave** [comment]
Thank you for taking the time to do this @mxochicale 

@fcooper8472 is definitely the expert here but overall LGTM

**Comment by NickleDave** [comment]
If there's no review by next meeting and I don't add to agenda please kindly refresh our memory

**Comment by NickleDave** [comment]
@fcooper8472 @tlestang friendly reminder to look at this in a little more detail when you get a chance

File: README.md [hunk 1]
```diff
@@ -1,52 +1,64 @@
...
+Web Server is available at http://localhost:1313/dev-review/ (bind address 127.0.0.1)
+Press Ctrl+C to stop
+```
+See [Launching the website locally (hugoloveit.com)](https://hugoloveit.com/theme-documentation-basics/#25-launching-the-website-locally) for further information. 
+
+## Contributing
+Commit changes to any other branch than `main` or `gh-pages`, open a pull request and request for reviews as common practice following the [pull request workflow with git](https://medium.com/@urna.hybesis/pull-request-workflow-with-git-6-steps-guide-3858e30b5fa4).  
+Once your changes are merged into `main` branch, the site will be automatically built and deployed and will be live roughly 1 minute later. 
+
+:warning: **Do not commit directly to `main` or `gh-pages`.** :warning:
```
Comment (tlestang) [inline README.md:64]: How about we make both remote branches push-able only for admins?

**Pull request merged**